README.md

```diff
@@ -915,14 +915,14 @@ The output is processed and displayed in the migration dashboard using the in `r
 
 ## [EXPERIMENTAL] Migration Progress Workflow
 
-The `migration-progress-experimental` workflow updates a subset of the inventory tables to track migration status of
-workspace resources that need to be migrated. Besides updating the inventory tables, this workflow tracks the migration
-progress by updating the following [UCX catalog](#create-ucx-catalog-command) tables:
+The manually triggered `migration-progress-experimental` workflow updates a **subset** of
+the [inventory tables](#assessment-workflow) to [track Unity Catalog compatability](docs/migration-progress.md) of Hive
+and workspace objects that need to be migrated.
 
```

Comment:
Describe the purpose and business purpose.

Reply:
I've added a reference to the dashboard, would like to cover your comment either in the dashboard documentation (shortly) and/or the migration progress documentation (more extensively), as this section is only about the workflow and less about the "why"